students/12th/sonbyungjin/js/main.js

```diff
@@ -0,0 +1,66 @@
+const thisIsInput = document.getElementById('commentInput');
+const thisIsButton = document.getElementsByTagName('button')[0];
+const thisIsUl = document.getElementsByTagName('ul')[0];
+let thisIsLike = '';

```

Comment:
```suggestion
const input = document.getElementById('commentInput');
const button = document.getElementsByTagName('button')[0];
const ul = document.getElementsByTagName('ul')[0];
let like = '';
```
변수 명에 thisIs를 붙이는 건 그다지 바람직해보이지 않네요. 어차피 무언가를 선언한다는 것은 `이것`에 대해 이야기하는 것이니까 당연한 소리인 것 같아서 인듯 합니다.